.github/workflows/ci.yml

```diff
@@ -35,9 +35,6 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
 
-      - name: use latest npm
```

Comment:
Looks like this was removed to get windows CI passing.

This file will eventually be managed buy `@npmcli/template-oss` and end up being clobbered the next time that module has a new version.

The ci.yml that template-oss installs works fine w/ windows so this change is also fine to have for now.

Reply:
:+1: